### PR TITLE
update azure_rm_appserviceplan related module documents 

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_appserviceplan related module documents 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_appserviceplan/azure_rm_appserviceplan_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
@Fred-sun thanks for the documentation updates. You need to get rid of the merge commits in this branch before we can consider it for merging. Can you [rebase](https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html#rebase-guide)? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_appserviceplan.py [hunk 1]
```diff
@@ -38,10 +38,10 @@
 
     sku:
         description:
-            - The pricing tiers, e.g., F1, D1, B1, B2, B3, S1, P1, P1V2 etc.
+            - The pricing tiers, e.g., C(F1), C(D1), C(B1), C(B2), C(B3), C(S1), C(P1), C(P1V2) etc.
             - Please see U(https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) for more detail.
             - For linux app service plan, please see U(https://azure.microsoft.com/en-us/pricing/details/app-service/linux/) for more detail.
-
+            - Sku needs to be configured at creation time.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_appserviceplan.py]: not necessary, user could update sku to scale up or down.

**Comment by Fred-sun** [comment]
> @Fred-sun thanks for the documentation updates. You need to get rid of the merge commits in this branch before we can consider it for merging. Can you rebase? Thanks!

@acozine Already rebase the branch. Thanks a lot!

File: lib/ansible/modules/cloud/azure/azure_rm_appserviceplan.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_appserviceplan.py]: Thanks for your review, already remove this line!

**Comment by Fred-sun** [comment]
ready_for_review

**Review by yuwzho — APPROVED** [review]
shipit

**Comment by acozine** [comment]
Thanks @Fred-sun and @yuwzho!

**Pull request merged**